go.mod

```diff
@@ -21,14 +21,14 @@ require (
 	golang.org/x/sync v0.3.0
```

Comment:
Can we make sure to add testing for the new logic so that we ensure that we are making the correct decisions based on this new logic? A good way to do this is to add tests to the `pkg/controllers/provisioning/nodepool_test.go` and `pkg/controllers/provisioning/provisioner_test.go` files that will validate that we handle restartable init containers correctly

Reply:
ya sure